pallets/creditcoin/src/lib.rs

```diff
@@ -447,8 +447,8 @@ pub mod pallet {
 		/// Only the lender can perform the action.
 		NotLender,
 
-		/// The queue of unverified transfers is full for this block.
-		UnverifiedTransferPoolFull,
+		/// The pool of unverified tasks pending verification is full.
+		UnverifiedTaskPoolFull,
```

Comment:
This will also require an update to the interface types in `creditcoin-js`. @jfrank-summit should this update be handled in the same PR? 

I think it should.